database/migrations/2014_10_12_000000_create_user_table.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateUserTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('user', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('employee_code')->unique();
+            $table->string('name');
+            $table->string('email')->unique();
+            $table->string('team');
+            $table->string('avatar_url')->nullable();
+            $table->tinyInteger('role')->default(0);            
```

Comment:
- you should add comment for role.
ex: $table->tinyInteger('role')->default(0)->comment="0: admin; 1:user"